src/shibuya.js

```diff
@@ -28,24 +28,24 @@ const main = async () => {
     const proxyOnShibuya = shibuyaHelper.getProxyAccount(TURING_PARA_ID, shibuyaAddress);
 
     console.log(`\na) Add a proxy of Turing (paraId:${TURING_PARA_ID}) for Alice on Shibuya ...\n Proxy address: ${proxyOnShibuya}\n`);
-    await sendExtrinsic(shibuyaHelper.api, shibuyaHelper.api.tx.proxy.addProxy(proxyOnShibuya, 'Any', 0), keyPair);
+    await sendExtrinsic(shibuyaHelper.api, shibuyaHelper.api.tx.proxy.addProxy(proxyOnShibuya, 'Any', 0), keyPair); 
 
     console.log('\nb) Topping up the proxy account on Shibuya with SBY ...\n');
```

Comment:
The top-up doesn’t need to run every time. Always check if the balance falls below MIN_BALANCE_IN_PROXY(defined on the top). If it is, transfer MIN_BALANCE_IN_PROXY amount to the proxy address.